presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -321,6 +323,16 @@ protected void assertTableColumnNames(String tableName, String... columnNames)
         assertEquals(actual, expected);
     }
 
+    protected void assertExplainAnalyze(@Language("SQL") String query)
```

Comment:
I think this method should be `@Deprecated` because it's not much better than
```
computeActual("EXPLAIN ANALYZE " + query); // ensure does not fail
```

yet, it pretends it is does something useful

Reply:
i do not think this is the right direction. I'd expect the test to do assertThat() and do validation as found fit in this particular case

Reply:
I'm pretty sure you would end up with similar helper method that would cut boilerplate

Reply:
https://github.com/prestosql/presto/pull/6139